engine/src/test/java/com/datatorrent/stram/cli/ApexCliTest.java

```diff
@@ -126,10 +130,14 @@ public void testLaunchAppPackagePropertyPrecedence() throws Exception
   @Test
   public void testLaunchAppPackageParametersWithConfigPackage() throws Exception
   {
-    ApexCli.LaunchCommandLineInfo commandLineInfo = ApexCli
-        .getLaunchCommandLineInfo(new String[]{"-exactMatch", "-conf", configFile.getAbsolutePath(), appFile.getAbsolutePath(), "MyFirstApplication"});
+    ApexCli.LaunchCommandLineInfo commandLineInfo =
+        getLaunchCommandLineInfo(new String[]{"-exactMatch", "-conf", configFile.getAbsolutePath(), appFile.getAbsolutePath(), "MyFirstApplication"});
+
+    commandLineInfo.args = new String[1];
+    commandLineInfo.args[0] = "MyFirstApplication";
```

Comment:
Please use the notation `new String[]{ "item1", "item2", ... }` instead of doing element assignment like this.